### PR TITLE
Minor: Refactor CommandQueue to accept PreparedStatements

### PR DESCRIPTION
**Pull request by big-andy-coates**
### Description 
The `CommandQueue` interface was created _before_ `PreparedStatement` existed.  Now that the latter exists it makes sense that the queue takes a single `PreparedStatement` in place of the current (`String`, `Statement`) pair.

### Testing done 
Non-functional change, so just running existing tests.

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java [hunk 1]
```diff
@@ -44,14 +46,14 @@
...
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.ArgumentMatchers.isA;
 import static org.mockito.Mockito.doAnswer;
 import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.inOrder;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
+import static org.mockito.hamcrest.MockitoHamcrest.argThat;
```
Comment (rodesai) [inline ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java]: TIL

Comment (agavra) [reply ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java]: 👍 this is so useful (just used it in a test I'm working on!)

**Comment by big-andy-coates** [comment]
test this please

**Comment by big-andy-coates** [comment]
retest this please

**Pull request merged**